spec/domain/people/merger_spec.rb

```diff
@@ -68,17 +68,393 @@
       person_roles = person.roles.with_inactive
       expect(person_roles.count).to eq(2)
       group_ids = person_roles.map(&:group_id)
-      expect(group_ids).to include(groups(:bottom_group_one_one).id)
+      expect(group_ids).to include(group.id)
       expect(group_ids).to include(groups(:bottom_group_two_one).id)
 
       expect(Person.where(id: duplicate.id)).not_to exist
     end
 
+    it "merges invoices" do
+      @source = duplicate
+      @target = person
+
+      target_invoice_1 = Fabricate(:invoice, group: group, recipient: person)
+      source_invoice_1 = Fabricate(:invoice, group: group, recipient: duplicate)
+      source_invoice_2 = Fabricate(:invoice, group: group, recipient: duplicate)
+
+      expect do
+        merger.merge!
+      end.to change(Person, :count).by(-1)
+        .and change(person.invoices, :count).by(2)
```

Comment:
besser fände ich hier `.and not_change { Invoice.count }` das andere wird ja noch mit den assertions weiter unten getestet, sollte wenn über alle specs angepasst werden